schemas/public/spec-plugins.json

```diff
@@ -87,6 +87,10 @@
               "type": "string"
             },
             "description": "The values of the release"
+          },
+          "disableValidationOnInstall": {
+            "type": "boolean",
+            "description": "Disable helm diff validation on install"
```

Comment:
```suggestion
            "description": "Disable running `helm diff` validation when installing the plugin, it will still be done when upgrading."
```